Shared/GameFiles/WsModel/WsModelMaterialFile.cs

```diff
@@ -53,6 +55,8 @@ public WsModelMaterialFile(PackFile pf, string fullPath)
 
                 if (textureSlotName == "s_diffuse")
                     Textures[TextureType.Diffuse] = texturePath;
```

Comment:
In general, when there is a lot of copy and paste, its a good idea to see if there is a pattern that can be extracted.
I think there is an easy pattern to extract here